src/collection-role/AuthManager.ts

```
@@ -124,3 +129,18 @@ export function isAdmin(user: UserToken): boolean {
 function isCurator(user: UserToken, collection: string): boolean {
   return user.accessGroups.includes(`${ROLES.CURATOR}@${collection}`);
 }
+
+/**
+ * Checks if request should be authorized by checking if `authorizationCases` contains `true`.
+ *
+ * @export
+ * @param {boolean[]} authorizationCases [List of boolean values from the result of an authorization check]
+ */
+export function authorizeRequest(authorizationCases: boolean[]): never | void {
```

What's the benefit of using never here?

---

To convey to maintainers that the function should throw an error. Closest thing to `throws` that Typescript has.